modules/oe_content_persistent/tests/Kernel/PersistentUrlTest.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_content_persistent\Kernel;
+
+use Drupal\KernelTests\KernelTestBase;
+use Drupal\language\Entity\ConfigurableLanguage;
+use Drupal\node\Entity\Node;
+use Drupal\node\Entity\NodeType;
+use Drupal\user\RoleInterface;
+
+/**
+ * Tests Persistent url related controller and service.
+ *
+ * @group path
+ */
+class PersistentUrlTest extends KernelTestBase {
+
+  /**
+   * Modules to enable.
+   *
+   * @var array
+   */
+  public static $modules = [
+    'path',
+    'node',
+    'user',
+    'system',
+    'language',
+    'content_translation',
+    'oe_content_persistent',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+
+    $this->installEntitySchema('node');
+    $this->installEntitySchema('user');
+    $this->installEntitySchema('configurable_language');
+    $this->installSchema('node', ['node_access']);
+    \Drupal::service('router.builder')->rebuild();
+
+    $node_type = NodeType::create(['type' => 'page']);
+    $node_type->save();
+
+    $this->installConfig(['language']);
+    $this->installConfig(['user']);
+
+    ConfigurableLanguage::create(['id' => 'fr'])->save();
+
+    $config = $this->config('language.negotiation');
+    $config->set('url.prefixes', ['en' => '', 'fr' => 'fr'])
+      ->save();
+
+    user_role_grant_permissions(RoleInterface::ANONYMOUS_ID, ['access content']);
+
+    \Drupal::service('kernel')->rebuildContainer();
+
+  }
+
+  /**
+   * Test return of ContentUuidResolver service.
+   */
+  public function testContentUuidResolver(): void {
+
+    /** @var \Drupal\oe_content_persistent\ContentUuidResolver $uuid_resolver */
+    $uuid_resolver = \Drupal::service('oe_content_persistent.resolver');
+
+    // @TODO Investigate impact.
+    /** @var \Drupal\Core\PathProcessor\PathProcessorManager $path_processor_manager */
+    $path_processor_manager = \Drupal::service('path_processor_manager');
```

Comment:
Is this path processor part still needed since we don't get the alias? I guess the point of this test is to test that we get the correct entity not that it has a correct alias. So we can remove this no?